react/week2/todo-app/src/Todo.js

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react';
+import todoList from "./TodoList";
+
+const Todo = () => {
+  const [todos, setTodos] = useState(todoList)
+  const [check,setCheck] = useState(false)
+  const newTodo = {id: todos.length+1, title:'random text'}
```

Comment:
Minor comment but this logic may lead to multiple items having the same id.

If I add an item, then remove another item further up & add a new one it would be at the same "length" & you may end up have multiple items with the same id when created

Reply:
done